messaging_components/clients/external/client_external.py

```diff
@@ -0,0 +1,92 @@
+from messaging_abstract.component import Client, Node, Executor
```

Comment:
I opened issue under [messaging_abstract #2](https://github.com/rh-messaging-qe/messaging_abstract/issues/2) for removal execution. So we should get execution layer back to components or somewhere else. It depends on the upcoming discussion.